tools/SetupFlow/DevHome.SetupFlow/Views/ReviewView.xaml

```diff
@@ -62,7 +62,8 @@
             <StackPanel>
                 <Expander HorizontalAlignment="Stretch" HorizontalContentAlignment="Stretch"
                           IsExpanded="true"
-                          MinHeight="64" CornerRadius="4" Padding="24,0,24,7" >
+                          MinHeight="64" CornerRadius="4" Padding="24,0,24,7"

```

Comment:
Does it let you use `ThemeResource ControlCornerRadius` for CornerRadius? That's what SettingsCard and SettingsExpander use, and looking at it I think what xaml Expander uses by default anyway.

Reply:
I totally can.